src/websocket.rs

```diff
@@ -31,7 +31,7 @@ impl AdminWebsocket {
     pub async fn generate_agent_pubkey(&mut self) -> Result<AgentPubKey> {
         let response = self.send(AdminRequest::GenerateAgentPubKey).await?;
         match response {
-            AdminResponse::GenerateAgentPubKey(agent_pubkey) => Ok(agent_pubkey),
+            AdminResponse::AgentPubKeyGenerated(agent_pubkey) => Ok(agent_pubkey),
```

Comment:
This was renamed in https://github.com/holochain/holochain/commit/949a2c9d064cef13390ad08f34619141cc01911d.